src/sidebar/templates/annotation-header.html

```diff
@@ -23,7 +23,11 @@
     <br>
     <span class="annotation-header__share-info">
       <a class="annotation-header__group"
-        target="_blank" ng-if="vm.group() && vm.group().url" href="{{vm.group().url}}">
+        ng-if="vm.group() && vm.group().type == 'open'">
+        <i class="h-icon-public"></i><span class="annotation-header__group-name">{{vm.group().name}}</span>
+      </a>
+      <a class="annotation-header__group"
+        target="_blank" ng-if="vm.group() && !vm.group().type == 'open' && vm.group().url" href="{{vm.group().url}}">
```

Comment:
I'll note that we don't need to change the behaviour here for open groups any more because activity pages works for them. However, I don't think this code does what you intended.

In `!vm.group().type == 'open'` the unary `!` operator has [higher precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) than the binary `==` operator so if the group type is "private" this will evaluate to `!'private' == open` => `false == 'open'` => `false` and hide the link instead of showing it.